### DISCUSSION TITLE
Having a Navbar and using Parrallax ?

### DISCUSSION THREAD

**Discussion by TropicalBottle** [pmndrs/react-spring#2023] [Support]
**Distinguished greetings !**
I tried to use some Parrallax with spring for a project and everything was doing okay until I saw that my Navigation menu wasn't working anymore. I searched for few days on stackoverflow and some forum and I see everyone having the same problem and nobody having a solution except going to another library..

**Did someone managed today to get a Navbar working in the same time as a Parrallax ?**
[
You can see the problem here](https://codesandbox.io/p/sandbox/intelligent-joji-jnr5jh?file=%2FREADME.md&workspace=%257B%2522activeFileId%2522%253A%2522cl8skx9az000llpge5yew53n5%2522%252C%2522openFiles%2522%253A%255B%2522%252FREADME.md%2522%255D%252C%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522sidekickItems%2522%253A%255B%257B%2522type%2522%253A%2522PREVIEW%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522key%2522%253A%2522clafi6w2a000n2a69ta9kpzft%2522%252C%2522isMinimized%2522%253Afalse%257D%255D%257D)

Thank you very much !

**Comment by joshuaellis** [comment]
Whilst i'm unable to offer help right now, i'd recommend making a small reproduction of the issues you're facing and share it here so people have a more accurate picture of the issues you're facing otherwise it's a little vague. Good luck in solving your issues! 😄

**Reply by TropicalBottle** [reply]
Yeah I'm actually trying to put that on my Cpanel server for hours but that's hell XD But I put it on CodeSandbow ! Thanks !
